pages/index.js

```diff
@@ -275,15 +282,21 @@ export default function Home({
           <div className="mt-16 mb-8 space-y-16">
             {featuresDetails.map((feature, featureIdx) => (
               <div
+                ref={featureRefs[featureIdx][0]}
                 key={feature.name}
-                className="flex flex-col-reverse lg:grid lg:grid-cols-12 lg:items-center lg:gap-x-8"
+                className="flex flex-col-reverse overflow-x-hidden lg:grid lg:grid-cols-12 lg:items-center lg:gap-x-8"
               >
                 <div
                   className={classNames(
                     featureIdx % 2 === 0
-                      ? "lg:col-start-1"
-                      : "lg:col-start-8 xl:col-start-9",
-                    "mt-6 lg:mt-0 lg:row-start-1 lg:col-span-5 xl:col-span-4",
+                    ? "lg:col-start-1 mt-6 lg:mt-0"
+                    : "lg:col-start-8 xl:col-start-9",
+                  featureRefs[featureIdx][1] && featureIdx % 2 === 0
+                    ? "animate-fade-right"
+                    : featureRefs[featureIdx][1] && featureIdx % 2 !== 0
+                    ? "animate-fade-left"
+                    : "",
+                  "opacity-0 lg:row-start-1 lg:col-span-5 xl:col-span-4"
```

Comment:
Thank you for making the changes, it is better but still difficult to read as indentation is not correct, if you add the Prettier plugin to VScode it will fix it all for you (btw this is also for other files in the PR)

Reply:
Yeah, I always use it in general, but I was encountering some issues with that. See : 

![image](https://github.com/EddieHubCommunity/BioDrop/assets/99136041/3692c75e-a5f3-4e75-a8cc-d1ff82fda17b)

![image](https://github.com/EddieHubCommunity/BioDrop/assets/99136041/9a44e85a-06b4-40ee-9184-1bb48aba776f)

Reply:
seems like a setup issue with your npm, maybe you used `sudo` previously? check the permissions are set to your current user